client/knowledge_base_search/src/containers/ResultContainer/ResultContainer.jsx

```diff
@@ -1,9 +1,36 @@
 import React from 'react';
 import classNames from 'classnames';
 import { string, number, node, oneOfType } from 'prop-types';
+import { Icon } from 'watson-react-components';
 import './styles.css';
 
-function ResultContainer({ resultText, resultRank, children }) {
+function ResultContainer({ resultText, resultRank, originalRank, children }) {
+
+  function getRankDisplay(resultRank, originalRank) {
+    if (resultRank < originalRank) {
+      return (
+        <div className="training_comparison--rank-down">
+          <Icon type="up" style={{
```

Comment:
you can probably just use the sass selectors to target this icon. something like
```
.training_comparison {
  &--rank {
    &--down svg {
      transform: scale(0.7, 0.7);
      fill: color('purple', 20);
    }
  }
}
```